### DISCUSSION TITLE
[2021.08.23] 상위 n개 레코드

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#16] [SQL]
[상위 n개 레코드](https://programmers.co.kr/learn/courses/30/lessons/59405)

**Comment by pyunji** [comment]
[상위 n개 레코드](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/pyunji/sql/PR_59405.sql)

**Comment by chaeyeon4u** [comment]
[상위 n개 레코드](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/chaeyeon/homework/SQL/PR_59405.sql)

**Comment by leechang4** [comment]
[상위 n개 레코드](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR_59405.sql)

**Comment by hyk339** [comment]
[solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59405.sql)
